docs/examples/quick-start/index.md

```diff
@@ -7,6 +7,18 @@ title: Quick Start Guide
 
 This step-by-step guide will quickly get you started on Leaflet basics, including setting up a Leaflet map, working with markers, polylines and popups, and dealing with events.
 
+### Overview
+This is an overview of the topics that will be covered by this guide.
+	*Preparing your page
+	*Setting up the map
+	*Markers, circles, and polygons
+	*Working with popups
+	*Dealing with events
+	
+### Quick Links
+	#### <a href="/reference.html">Documentation</a>
+	#### <a href="../../examples.html">Tutorials</a>
```

Comment:
While the Overview may be helpful, these Quick Links are already linked to and clearly visible at the top of all pages, is this really needed?